src/app/inner-pages/shared/components/job-full/job-full.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { getJobs, State } from 'app/core/reducers';
+import { Job } from 'app/shared/models';
+import { JobsService } from 'app/shared/services';
+import { Subscription } from 'rxjs';
+import { flatMap } from 'rxjs/operators';
+
+export enum JobSections {
+  Project,
+  Duration,
+  KeySkills
+}
+
+
+@Component({
+  selector: 'app-job-full',
+  templateUrl: './job-full.component.html',
+  styleUrls: ['./job-full.component.scss']
+})
+export class JobFullComponent implements OnInit, OnDestroy {
+  jobSubscribtion: Subscription = new Subscription();
+  job: Job;
+  JobSections = JobSections;
+  activeSection = JobSections.Project;
+
+  constructor(private store: Store<State>,
+              private route: ActivatedRoute,
+              private jobsService: JobsService
+  ) { }
+
+
+
+  ngOnInit(): void {
+    const jobId = this.route.snapshot.params.id;
+    this.jobSubscribtion = this.store.select(getJobs).pipe(
+      flatMap(jobs => {
```

Comment:
вынести в приватный метод
```suggestion
      flatMap(jobs => this.getJobFromStore(jobs)
```